examples/daal/cpp_sycl/source/utils/service.h

```diff
@@ -475,7 +475,7 @@ void printPackedNumericTable(NumericTable & dataTable, size_t nFeatures, const c
     printPackedNumericTable(&dataTable, nFeatures, message);
 }
 
-template <typename type1, typename type2>
+template <typename type1, typename type2, int prec1 = 3, int prec2 = 0>
```

Comment:
Is it really necessary? Why not as a function argument? It doesn't look good.

Reply:
Yes. It's necessary. Distances in normalized data are always less than 1 and they are rounded to 0.

Reply:
NA